pubsub/kafka/kafka.go

```diff
@@ -277,8 +290,20 @@ func (k *Kafka) Subscribe(req pubsub.SubscribeRequest, handler pubsub.Handler) e
 func (k *Kafka) getKafkaMetadata(metadata pubsub.Metadata) (*kafkaMetadata, error) {
 	meta := kafkaMetadata{}
 	// use the runtimeConfig.ID as the consumer group so that each dapr runtime creates its own consumergroup
-	meta.ConsumerID = metadata.Properties["consumerID"]
-	k.logger.Debugf("Using %s as ConsumerGroup name", meta.ConsumerID)
+	if val, ok := metadata.Properties["consumerID"]; ok && val != "" {
+		meta.ConsumerID = val
```

Comment:
Could you assign this to `meta.ConsumerGroup` instead and remove the need for `meta.ConsumerID`?

Reply:
I misunderstood your original statement and I correct it now